app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,8 @@
+class QuestionsController < ApplicationController
+  def index
+    @questions = Question.all_by_query_params(params)
+    respond_to do |format|
```

Comment:
Yes we can do this way also render json: @questions or we can use respond_to :json on top in the controller